### PR TITLE
Make 24h TimePicker more responsive on larger screens.

### PR DESCRIPTION
**Pull request by ssancho14**
#### Checklist :clipboard:
- [ ] Add explicit visibility modifier and explicit return types for public declarations
- [ ] Run spotless check
- [ ] Run tests
- [ ] Update metalava's signature text files

### CODE REVIEW

File: sample/src/main/java/com/google/android/horologist/scratch/PickerActivity.kt [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+        setContent {
+            WearPickerApp()
+        }
+    }
+}
+
+@Composable
+fun WearPickerApp() {
+    val sizes = listOf(190, 200, 210, 220, 230, 240, 250)
```
Comment (yschimke) [inline sample/src/main/java/com/google/android/horologist/scratch/PickerActivity.kt:85]:
```suggestion
    val sizes = listOf(190, 200, 210, 220, 230)
```

**Review by yschimke — APPROVED** [review]
Waiting for screenshots

File: composables/src/main/java/com/google/android/horologist/composables/TimePicker.kt [hunk 2]
```diff
@@ -138,13 +143,33 @@ public fun TimePicker(
...
+    val isLargeScreen = LocalConfiguration.current.screenWidthDp > 225
     val textStyle = with(LocalDensity.current) {
-        if (showSeconds) {
-            fontScaleIndependent(MaterialTheme.typography.display3)
-        } else {
-            fontScaleIndependent(MaterialTheme.typography.display1)
-        }
+        fontScaleIndependent(
+            when {
+                !showSeconds -> MaterialTheme.typography.display1
```
Comment (BowerSteve) [inline composables/src/main/java/com/google/android/horologist/composables/TimePicker.kt:150]: Not sure the display1, display2, display3 match the spec...

**Review by BowerSteve — COMMENTED** [review]
TimePickerWith12HourClock needs to be updated too

File: composables/src/main/java/com/google/android/horologist/composables/TimePicker.kt (hunk 2 above)
Comment (ssancho14) [reply composables/src/main/java/com/google/android/horologist/composables/TimePicker.kt:150]: I see no spec for hour + minutes picker... display1 is what it was before. Happy to use something else if we have a better alternative.

Well, there is no display4 on our theme, and current constants do match the sp sizes specified in the spec.

File: sample/src/main/java/com/google/android/horologist/scratch/PickerActivity.kt (hunk 1 above)
Comment (ssancho14) [reply sample/src/main/java/com/google/android/horologist/scratch/PickerActivity.kt:85]: Done.

**Review by yschimke — APPROVED** [review]

**Pull request merged**